src/main/java/io/opentracing/contrib/global/GlobalTracer.java

```diff
@@ -38,43 +38,26 @@
  * The {@linkplain GlobalTracer} will not attempt to choose between implementations:</li>
  * <li>If no single tracer service is found, the {@link io.opentracing.NoopTracer NoopTracer} will be used.</li>
  * </ol>
+ *
+ * @see io.opentracing.util.GlobalTracer
+ * @deprecated GlobalTracer is now part of OpenTracing-java
  */
-public final class GlobalTracer implements Tracer {
```

Comment:
Fair point about the class signature, fortunately the API surface should've been static method calls mostly (and calls to `Tracer` API).
I really l would like a legacy provide one more ~~`GlobalTracer.get()`~~ implementaiton to return the new GlobalTracer instance.

I am willing to remove the reflection code and 'break' the current `register()` compatibility if there are others objecting to that. I'm actually also in doubt myself about the reflection code in there.